src/utils/RNPxlSearch.cpp

```diff
@@ -188,6 +188,13 @@ class RNPxlSearch :
                         false, 
                         false);
 
+    registerStringList_("RNPxl:nt_groups",
+        "",
+        {},
+	"Restrict which nucleotides can cooccur in a precursor adduct to be able to search both RNA and DNA (Formate e.g.: AU CG).",

```

Comment:
can you explain what for example "AU CG" would actually do? I am still not sure what it does, you then search for "AU" and "CG" in the result?

Reply:
No it means that precursor adduced may be composed of As and Us or alternatively Cs and Gs. But no mix between both e.g. AAC-H2O